source/excerptexport/urls.py

```diff
@@ -1,23 +1,21 @@
 from django.conf.urls import patterns, url
 
-from excerptexport import views
-
 
 urlpatterns = patterns(
-    '',
-    url(r'^$', views.index, name='index'),
```

Comment:
I'm not a fan of 'magic' string references instead of real references. What's the reason using string references? Circular references? Can't python handle this?